include/cantera/transport/Transport.h

```diff
@@ -781,19 +792,22 @@ class Transport
     ThermoPhase* m_thermo;
 
     //! true if finalize has been called
+    //! @deprecated To be removed after Cantera 3.0
     bool m_ready = false;
 
     //! Number of species
     size_t m_nsp = 0;
 
     //! Number of dimensions used in flux expressions
+    //! @deprecated To be removed after Cantera 3.0
     size_t m_nDim;
 
     //! Velocity basis from which diffusion velocities are computed.
     //! Defaults to the mass averaged basis = -2
     int m_velocityBasis = VB_MASSAVG;
 
     //! reference to Solution
```

Comment:
This isn’t currently used, but could be useful once `Solution` is used consistently. It’s still ok to deprecate though.